src/main/java/com/denizenscript/clientizen/events/EntityStartsStopsRenderingScriptEvent.java

```diff
@@ -0,0 +1,58 @@
+package com.denizenscript.clientizen.events;
+
+import com.denizenscript.clientizen.objects.EntityTag;
+import com.denizenscript.denizencore.events.ScriptEvent;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import net.minecraft.entity.Entity;
+
+public class EntityStartsStopsRenderingScriptEvent extends ScriptEvent {
+
+    // <--[event]
+    // @Events
+    // <entity> starts|stops rendering
+    //
+    // @Triggers when an entity is rendered by the client's camera. This does not mean the entity will always be visible, but within the camera's viewing frustum.
```

Comment:
It isn't really being rendered by the camera, just based on it (which the next bit explains well, so can probably just make this `by the client`?)